data/report_templates/project_summary.md

```diff
@@ -160,9 +184,8 @@ UPPNEX project, which was created for you when your order was placed:
 {{ project.UPPMAX_path }}
 ```
 
-
-If you have problems accessing your data, please contact SciLifeLab
-[genomics_support@scilifelab.se](mailto: genomics_support@scilifelab.se).
+If you have problems accessing your data, please contact NGI
+[{{ project.support_email }}](mailto: {{ project.support_email }}).
```

Comment:
Not sure if it'll be happy with the space between `mailto:` and `{{ project.support_email }}`? Might be fine of course.